src/Views/InitializationPage.xaml.cs

```diff
@@ -23,8 +23,8 @@ public InitializationPage(InitializationViewModel initializationViewModel)
         ViewModel = initializationViewModel;
     }
 
-    private async void Page_Loaded(object sender, RoutedEventArgs e)
+    private void Page_Loaded(object sender, RoutedEventArgs e)
     {
-        await ViewModel.OnPageLoadedAsync();
+        ViewModel.OnPageLoaded();

```

Comment:
Does OnPageLoadedAsync() get called from anywhere anymore? If not, delete.

Reply:
No, but it contains the code we used before to install the extensions. I was thinking on keeping it here as this can be used again in the future. Should we still remove it if this is the case?

Reply:
How about putting it in a /* */ comment with an additional comment about what it does and that we're leaving it here in case it's useful in the future. We could take it out entirely since it would still be in github history but I understand the desire not to.

Reply:
Ended up removing it. I will keep in mind that this existed if we need in the future.